shared/types/guards.ts

```diff
@@ -83,3 +84,41 @@ export const isWalletFile = (wallet: any): wallet is WalletFile =>
   isWalletMeta(wallet.meta) &&
   (isWalletGCMEncrypted(wallet.crypto) ||
     isWalletLegacyEncrypted(wallet.crypto));
+
+export const validationWalletCipherTextDuplication = (
```

Comment:
It is not a type predicate (previously known as "type guard"):
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates